src/framework/theme/components/popover/helpers/adjustment.helper.ts

```diff
@@ -26,8 +26,13 @@ const NB_ORDERED_PLACEMENTS = {
 @Injectable()
 export class NbAdjustmentHelper {
 
-  constructor(private positioningHelper: NbPositioningHelper, private window: NbWindow) {
-  }
+  private window: Window;
+
+  constructor(
+    private positioningHelper: NbPositioningHelper,
+    @Inject(NB_WINDOW) window) {
+      this.window = window as Window;
```

Comment:
According to [this changelog](https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#new-tsx-file-extension-and-as-operator) **as** operator was introduced to support **.tsx**. Maybe we have to use basic type cast instead: **this.window = \<Window\> window**.
@nnixaa, do you think?